src/myr/Myr.js

```diff
@@ -314,14 +314,22 @@ class Myr {
     return outerElId;
   }
 
-  // Disallows the entity to be pushed
-  makeUnPushable = (outerElId) => {
```

Comment:
Why are we taking out makeUnPushable?

Reply:
that was an accident which should be fixed now